e2e/init/init_test.go

```diff
@@ -85,7 +85,7 @@ var _ = Describe("init flow", func() {
 			Expect(app.Routes[0].Environment).To(Equal("test"))
 			Expect(app.Routes[0].URL).To(HaveSuffix(appName))
 			Eventually(func() (int, error) {
-				resp, fetchErr := http.Get(fmt.Sprintf("http://%s/", app.Routes[0].URL))
+				resp, fetchErr := http.Get(app.Routes[0].URL)
```

Comment:
Can you also make this modification to the `addons/` e2e tests 😅
1. https://github.com/aws/amazon-ecs-cli-v2/blob/ae84b9cbc216c0684469efc10d11d3250600d155/e2e/addons/addons_test.go#L182
2. https://github.com/aws/amazon-ecs-cli-v2/blob/ae84b9cbc216c0684469efc10d11d3250600d155/e2e/addons/addons_test.go#L162

Reply:
Yup. Took `fmt.Sprintf` out of 182 and `http://` out of 162.